dist/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To-Do List</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+        integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous" />
+<script defer src="main.bundle.js"></script></head>
+
+<body>
+    <div class="container">
+        <h4 class="title">Today's Tasks</h4>
+        <form>
+            <input class="input-todo" type="text" placeholder="Add to your List..." autofocus />
+            <button class="add-btn">
+                <i class="fas fa-location-arrow"></i>
+            </button>
+        </form>
+        <div class="todos">
+        </div>
+        <div class="clear">
+            <a>Clear All Completed</a>
+        </div>
+    </div>
```

Comment:
I recommend you minimise the use of div tags and use the html recommended semantic tags e.g section, main, nav and e.t.c

Reply:
Thank you for your feedback.